crates/types/src/message.rs

```diff
@@ -82,7 +82,7 @@ pub enum MessagePurpose {
     /// Message with a view sync certificate.
     ViewSyncCertificate,
     /// Message with a DAC.
-    DAC,
+    Dac,
```

Comment:
I was going to suggest `DaCertificate` by analogy with the others, but after a quick search I'm wondering if we even use these. Was this just for the web server?

Reply:
Yeah looks like the `Dac` variant isn't used. I'm wondering if we can actually remove the `MessagePurpose` enum and probably some other relevant types.

I just made an issue for this: https://github.com/EspressoSystems/HotShot/issues/3179. In this PR I just renamed it to `DaCertificate` which I think is a good suggestion!